src/DynamoCore/Graph/Connectors/ConnectorModel.cs

```diff
@@ -32,7 +32,20 @@ public enum ConnectorType
     public class ConnectorModel : ModelBase
     {
         #region properties
+        private bool isVisible = true;
 
+        /// <summary>
+        /// IsVisible flag controlling the visibility of a connector
+        /// </summary>
+        public bool IsVisible
```

Comment:
I think @SHKnudsen will introduce a property called `IsCollapsed` in the group improvements work, you can refer to https://github.com/DynamoDS/Dynamo/pull/12019/files. I feel maybe we need to align here?

Reply:
Hi @QilongTang the `IsCollapsed` property is on the ViewModelBase and this is on the `ConnectorModel` i believe this was needed to be able to add it to the undo/redo stack. We couldnt find a way to add view properties to be part of undo/redo @M-JULIANI can properly chime in on this

Reply:
Yeah that's right, when a property is modified we needed it to be in the model. Commands can be stacked on `undo/redo` more easily. When this flag changes, it is relayed over the the `ConnectorViewModel` which in turn controls the opacity of the connector. Happy to rename it for clarity.

Reply:
Ended up changing `IsVisible` to `IsDisplayed` and `IsPartlyVisible` to `IsTermporarilyDisplayed`.